.github/workflows/publish-to-dockerhub.yml

```diff
@@ -29,15 +28,17 @@ jobs:
       - name: Get tags
         run: git fetch --tags origin
 
-      - name: Login to DockerHub
+      - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_TOKEN }}
+          registry: ghcr.io
+          username: ${{ secrets.GH_USERNAME }}
+          password: ${{ secrets.GH_PAT}}
+
       - name: Push image
         env:
-          DOCKERHUB_ACCOUNT: ${{ secrets.DOCKERHUB_ACCOUNT }}
-          DOCKERHUB_REPO: ${{ secrets.DOCKERHUB_REPO }}
+          GITHUB_ACCOUNT: ${{ secrets.GITHUB_ACCOUNT }}
```

Comment:
Would it be possible to somehow infer the github account name from the variables the this action is triggered with? I imagine the `opencrvs` is available somewhere when the repository the pipeline is in is `opencrvs/opencrvs-countryconfig`

Reply:
Yeah, it's possible to use the default environment variables and infer the github account rather than hard-coding it! Let me refactor!